### PR TITLE
Implement GetLocalToGlobalTransform() in gRPC service

### PR DESCRIPTION
**Pull request by cschuet**

### CODE REVIEW

File: cartographer_grpc/mapping/pose_graph_stub.cc [hunk 1]
```diff
@@ -43,7 +43,12 @@ PoseGraphStub::GetAllSubmapPoses() {
 
 cartographer::transform::Rigid3d PoseGraphStub::GetLocalToGlobalTransform(
     int trajectory_id) {
-  LOG(FATAL) << "Not implemented";
+  grpc::ClientContext client_context;
+  proto::GetLocalToGlobalTransformRequest request;
```
Comment (gaschler) [inline cartographer_grpc/mapping/pose_graph_stub.cc]: write trajectory_id into request

Comment (cschuet) [reply cartographer_grpc/mapping/pose_graph_stub.cc]: Done.

**Comment by cschuet** [comment]
@wally-the-cartographer merge

**Comment by wally-the-cartographer** [comment]
Merge requested by authorized user cschuet. Merge queue now has a length of 1.

**Pull request merged**